action.yml

```diff
@@ -109,31 +129,30 @@ runs:
       shell: bash
       working-directory: ${{ inputs.working-directory }}
       run: |
-        resolver_nightly=
-
-        # Extra arguments for build and test steps, not the dependencies step
-        stack_build_arguments=()
-
-        if ${{ inputs.fast }}; then
-          stack_build_arguments+=( --fast )
-        fi
-
-        if ${{ inputs.pedantic }}; then
-          stack_build_arguments+=( --pedantic )
-        fi
+        {
+          echo "stack-query-arguments=              $(echo "${{ inputs.query-arguments       }}                                                 " | tr '\n' ' ')"
+          echo "stack-path-arguments=               $(echo "${{ inputs.path-arguments        }}                                                 " | tr '\n' ' ')"
+          echo "stack-setup-arguments=              $(echo "${{ inputs.setup-arguments       }}                                                 " | tr '\n' ' ')"
```

Comment:
Is GitHub doing some kind of whitespace trimming or is this actually setting things like

```sh
stack-query-arguments="                            --some-options                                "
```

?

I like the vertical alignment, but not if it's introducing actual white-space in the variables.

Reply:
As far as I know, there's no whitespace trimming. So yeah, the interpolation should be smarter.

Reply:
fixed in https://github.com/freckle/stack-action/pull/35/commits/fc83b2b156873472535e5ab335505e11e7270f39